crates/binjs_shared/src/ast.rs

```diff
@@ -261,3 +270,4 @@ pub trait Node: downcast_rs::Downcast {
     fn name(&self) -> &'static str;
 }
 impl_downcast!(Node);
+
```

Comment:
remove this maybe?

Reply:
The newline?

Reply:
yes, extra newline at the end of file.
if the diff is shown correctly, now the file ends with `\n\n`, but it should end with `\n`

Reply:
Actually, this gets rid of a missing newline, so I think I'll keep it :)